examples/oval.rb

```diff
@@ -8,17 +8,17 @@
   title: "Schwad's unbelievable desktop application that renders an oval",
   height: 700,
 ) do
-  flow do
+  flow(height: 200) do
```

Comment:
In theory we'd want shape do to be able to capture the ovals right? and render separately? but I'm sure that's out of scope right now

Reply:
I'm not sure. It's weird, because I think Shoes3 doesn't capture the ovals either. But then I'm not sure how that's supposed to happen. This is one of those places where either I don't understand what Shoes was trying to do, or it was poorly designed (the old one, not just us.) Hard to tell which yet, but I'll keep trying to figure it out.

Reply:
You *can* do this in Shoes3 by wrapping an image *around* the ovals and picking a size/resolution. But it'd be weird if that were the only way to do it.